File: SimpleSorting/ss.rb
# simple sorting soln in rb for codeeval.com by steven a dunn

File.open(ARGV[0], 'r').each_line do |line|
  line = line.chomp.split(' ').map {|sym| sym.to_f }
  line.sort!
  line.each do |elem|
    printf("%.3f ", elem)
  end
  puts
end

File: SimpleSorting/py3/ss.py
# Simple Sorting solution in Python 3 for CodeEval.com by Steven A. Dunn

import sys
from decimal import Decimal

for line in open(sys.argv[1], 'r'):
  line = line.rstrip('\n').split(' ')
  line = list(map(Decimal, line))
  line.sort()
  line = list(map(str, line))
  for i in line:
    sys.stdout.write("%s " % i),
  print()
